spec/presenters/summary/sections/case_details_spec.rb

```diff
@@ -125,6 +131,116 @@
       end
     end
 
+    context "when is_preorder_work_claimed=='no'" do
+      let(:has_case_concluded) { 'no' }
+      let(:date_case_concluded) { nil }
+
+      let(:is_preorder_work_claimed) { 'no' }
+      let(:preorder_work_date) { nil }
+      let(:preorder_work_details) { nil }
+
+      let(:is_client_remanded) { 'no' }
+      let(:date_client_remanded) { nil }
+
+      it 'has the correct rows' do
+        expect(answers.count).to eq(5)
+
+        answer = answers[0]
+        expect(answer).to be_an_instance_of(Summary::Components::ValueAnswer)
+        expect(answer.question).to eq(:case_type)
+        expect(answer.change_path).to match('applications/12345/steps/client/case_type')
+        expect(answer.value).to eq('foobar')
+
+        answer = answers[1]
+        expect(answer).to be_an_instance_of(Summary::Components::FreeTextAnswer)
+        expect(answer.question).to eq(:case_urn)
+        expect(answer.change_path).to match('applications/12345/steps/case/urn')
+        expect(answer.value).to eq('xyz')
+
+        answer = answers[2]
+        expect(answer).to be_an_instance_of(Summary::Components::ValueAnswer)
+        expect(answer.question).to eq(:has_case_concluded)
+        expect(answer.change_path).to match('applications/12345/steps/case/has_the_case_concluded')
+        expect(answer.value).to eq('no')
+
+        answer = answers[3]
+        expect(answer).to be_an_instance_of(Summary::Components::ValueAnswer)
+        expect(answer.question).to eq(:is_preorder_work_claimed)
+        expect(answer.change_path).to match('applications/12345/steps/case/claim_pre_order_work')
+        expect(answer.value).to eq('no')
+
+        answer = answers[4]
+        expect(answer).to be_an_instance_of(Summary::Components::ValueAnswer)
+        expect(answer.question).to eq(:is_client_remanded)
+        expect(answer.change_path).to match('applications/12345/steps/case/has_court_remanded_client_in_custody')
+        expect(answer.value).to eq('no')
```

Comment:
Do we need to check all the rows here? I think we only need to test the claim pre order work row
Same comment for the spec below too

Reply:
Make sense 👍